src/hypofuzz/frontend/src/tyche/Representation.tsx

```diff
@@ -1,18 +1,22 @@
-import { useEffect, useRef } from "react"
+import { useEffect, useRef, useState } from "react"
 import hljs from "highlight.js/lib/core"
 import "highlight.js/styles/github.css"
 import python from "highlight.js/lib/languages/python"
 import { Observation } from "../types/dashboard"
 import { TycheSection } from "./TycheSection"
+import { Pagination } from "../components/Pagination"
 
 hljs.registerLanguage("python", python)
 
 interface Props {
   observations: Observation[]
 }
 
+const perPage = 30
```

Comment:
Maybe 100?

Reply:
perPage would have to be dynamic based on observation text size if we did this...I'm on an m1 macbook pro and was getting ~500ms delay on 50 per page for larger reprs, it's still somewhat noticeable on e.g. https://hypofuzz.com/example-dashboard/#/tests/tests%2Ftest_provider.py%3A%3Atest_drawing_prefix_exactly. It's not even the syntax highlighting (I tested), putting that much text in the dom is just expensive. People are going to have even crazier lager reprs. Maybe we should truncate at some point and require clicking to expand?

Reply:
never mind, we can revisit later if a user complains.